.travis.yml

```diff
@@ -53,9 +53,21 @@ jobs:
         - PATH="$HOME/miniconda/bin:$PATH"
       install: .travis/install_conda.sh "Miniconda3-latest-Linux-x86_64.sh"
       script:
-        - .travis/conda_build_and_deploy.sh
+        - .travis/conda_build_and_deploy.sh --label 'dev'
         - .travis/trigger_docker_build.sh
         - .travis/trigger_readthedocs_build.sh
+    - stage: "Release"
```

Comment:
This entire section should be further down the list, otherwise the OSX builds will appear in the "Release" stage.

(why anyone ever thought this was a good idea is beyond me :man_facepalming:)